index.html

```diff
@@ -2405,7 +2405,31 @@ <h3>Zero-Knowledge Proofs</h3>
 has granted a degree to you without revealing your identity or any other
 personally identifiable information contained on the degree.
         </p>
-
+<p>
```

Comment:
Style nits: Properly indent this `<p>`, add  indented `</p>` below next line, indent the `<ol>`, remove `</p>` at bottom of this change.